### ISSUE TITLE
Rgb cathode light

### ISSUE THREAD

**Issue by MrHushPPG** [studio-minus/ppg-bugs-and-requests#1708]
Feature description
how about making rgb cathode light
Feature purpose
this cathode light would replace the rest

**Comment by DindinYT37** [comment]
Yeah but I think when choosing the color it should just be the colors we have right now and an "Advanced" button that opens the RGB slider/circle.

**Comment by MrHushPPG** [comment]
> Yeah but I think when choosing the color it should just be the colors we have right now and an "Advanced" button that opens the RGB slider/circle.

that's what I meant
